spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8
```

Comment:
> They used to be necessary in python2, but utf-8 is the default in python3. 

[Indeed!](https://peps.python.org/pep-3120/)

I imagine this is a holdover from SCT's Python2 days, and was left behind after the transition to Python3. 

I'm guilty of this too -- I've definitely been copying and pasting the file header declarations into new modules without really thinking about it? Basically treating old modules as templates for new modules. (And, from the git blames on various files, it seems like this is what other SCT devs have done too.)

Thank you for taking the time to stop and question whether these declarations are even necessary. :)

Reply:
> Remove the utf-8 coding declaration from [the CLI template on the dev wiki](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Programming%3A-CLI-script-structure#1-setup-description-and-attribution).

Done!